extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -164,7 +164,7 @@ function Ach_GetHTMLAfterBody($skin, &$html) {
 	global $wgOut, $wgTitle, $wgUser;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgTitle" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)